src/main/java/org/plumelib/util/RandomSelector.java

```diff
@@ -123,6 +123,7 @@ public RandomSelector(double keepProbability, Random r) {
    *
    * @param next value to be added to this selector
    */
+  @SuppressWarnings("index:argument")
```

Comment:
Every `@SuppressWarnings` annotation must have a comment explaining why the warning that it suppresses is a false positive.